go.mod

```diff
@@ -81,7 +81,7 @@ require (
 	github.com/smartcontractkit/chainlink-ccip/chains/solana v0.0.0-20250411163110-21a13ceb3ac4
 	github.com/smartcontractkit/chainlink-common v0.7.1-0.20250418172423-6b24a042d134
 	github.com/smartcontractkit/chainlink-data-streams v0.1.1-0.20250417193446-eeb0a7d1e049
-	github.com/smartcontractkit/chainlink-evm v0.0.0-20250418155716-b70a99cec060
+	github.com/smartcontractkit/chainlink-evm v0.0.0-20250425122252-68a01140308b
```

Comment:
Can we add a warning comment to the go.mod that this must not be changes on the release branch?

Reply:
How about a `replace github.com/smartcontractkit/chainlink-evm => github.com/smartcontractkit/chainlink-evm v0.0.0-20250425122252-68a01140308b`? (also with a comment?)
Then it can't be indirectly bumped on accident.